### PR TITLE
Final details

### PR DESCRIPTION
**Pull request by sz16900**
## Completed [Project #5 - Smashing Magazine](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown) in partial requirements of the [Microverse Curriculum](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/).
---
### Contributors
- Seth Zea
- Simon Peter Debbarma
--- 
##### General Requirements
- [x] Add descriptive README file to your repository.
- [x] Use meaningful names for your Pull Requests and add a short summary of its content.
- [x] Use descriptive commit messages.
- [x] Do not commit files that are not related to the project. Instead, add them to .gitignore file.
- [x] Do not commit inline comments. 
##### Other Requirements
- [x] Keep your code clean.
- [x] Remember to use semantic HTML.
- [x] Validate your HTML markup.
- [x] Use stylelint to link CSS files.
##### Project Specific
- [x] Align all elements with float OR flex AND your own Grid as described in previous lesson 
- [x] Use article tag for all articles listed on the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+        <div class="top-box">Smashing Magazine</div>
+        <div class="top-box">Articles</div>
+        <div class="top-box">Books</div>
+        <div class="top-box">Events</div>
+        <div class="top-box">Jobs</div>
+        <div class="top-box">Membership</div>
+        <div class="top-box">Topic</div>
+    </header>
+    <div class="feature-articles">
+        <!-- <div class="dont-miss">lorem bla</div> -->
```
Comment (pelzolga123) [inline index.html]: Commented code should be removed.

**Comment by pelzolga123** [comment]
## Hi Team!  
### You did a great job! :1st_place_medal: 
### Kudos for descriptive Pull Request :+1: 
Here a few things to change, before project can be approved:

* [ ] This part is missing:
image

* [ ] Footer should have centered links. Right now you just have `li` without links and they are not displayed like on original website.  Example:
image

* [ ] 1 article row should contain 1 article, which takes full width. `book-ad2`, `book-ad` should be articles. The same rule applies for 2 article row and 3 article row. Please change `div` to `article` where needed. [Link to page] (https://www.smashingmagazine.com/)

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by simonpeterdebbarma** [comment]
# @pelzolga123 Thanks for the review. 👌
---

Allow me to address the issues raised. 

1. If you are talking about the missing article, in the first section (i.e.: .suggested-articles.), it might be because you're on a screen that's between 1440px and 2200px wide. We have emulated the responsiveness of the original website. They behave differently, and show 4 articles on smaller (2 rows), or larger screens (1 row), similar to the original site.

2. The section you show isn't visible on us. [Here's a screenshot of what I see.](https://imgur.com/NAzyJw5.png) We do not know where it goes on the page.

3. We shall fix the footer links as requested.

4.  We had ignored the ads, and promos on the website as they are not articles. But we are making the changes as requested. 

5. Please clarify this requirement - `1 article row should contain 1 article, which takes full width`

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project._ *Happy coding!*

**Pull request merged**